main.go

```diff
@@ -13,8 +13,11 @@ import (
 
 func main() {
 	// Read config file and generate mDNS forwarding maps
-	configPath := flag.String("config", "", "Config file in TOML format")
+	configPath := flag.String("config", "", "Config file in TOML format (if not provided, ./config.toml is used)")
 	flag.Parse()
+	if *configPath == "" {
```

Comment:
It's actually simpler to use the second argument of [`flag.String()`](https://golang.org/pkg/flag/#String), it is the default value 😄